cpp/tools/clustertool.h

```diff
@@ -10,9 +18,15 @@ class ClusterTool {
   // Initialise a fresh log cluster:
   //  - Creates /serving_sth containing a new STH of size zero
   //  - Creates the /cluster_config entry.
-  static util::Status InitLog(TreeSigner<Logged>* tree_signer,
+  static util::Status InitLog(const ct::ClusterConfig& cluster_config,
+                              TreeSigner<Logged>* tree_signer,
                               ConsistentStore<Logged>* consistent_store);
 
+  // Sets the cluster config
+  static util::Status SetClusterConfig(
+      const ct::ClusterConfig& cluster_config,
+      ConsistentStore<Logged>* consistent_store);
```

Comment:
Yeah, fair point - I've removed their shackles and they're free to gambol through the `.text SECTION` as they please.